tests/core/test_strftime.cpp

```diff
@@ -140,6 +140,48 @@ int main() {
   size = strftime(s, 10, "%g", &tm);
   test((size == 2) && !cmp(s, "98"), "strftime test #29", s);
 
+  tm.tm_wday = 0;
+  size = strftime(s, 2, "%w", &tm);
+  test((size == 1) && !cmp(s, "0"), "strftime test #30", s);
```

Comment:
These new tests don't seem to be testing the new format specifiers (e.g. `%EY`).   Maybe I'm missing something?

Reply:
Actually I've focused on test to check `%w` as was pointed out in https://github.com/emscripten-core/emscripten/pull/8630#discussion_r285136190

I'll add more tests new format specifiers.